pkg/types/interfacetests/utils.go

```diff
@@ -20,11 +20,30 @@ type BasicTester[T any] interface {
 	Setup(t T)
 	Name() string
 	GetAccountBytes(i int) []byte
+	IsDisabled(testID string) bool
+	DisableTests(testIDs []string)
 }
 
-type testcase[T any] struct {
-	name string
-	test func(t T)
+type TestSelectionSupport struct {
+	enabledTests map[string]bool
```

Comment:
maybe its more intuitive if we call this disabledTests, since the default is enabled?